src/main/java/cloud/fogbow/ras/core/models/orders/ComputeOrder.java

```diff
@@ -41,7 +38,7 @@
     @Embedded
     private ArrayList<UserData> userData;
 
-    @Size(max = Order.FIELDS_MAX_SIZE)
+    @Size(max = 4)
```

Comment:
I believe this size was fixed in 4 while debugging. Please have a look if this size should be fixed.

Reply:
Yes, setting it back.